pages/item/_itemId/index.vue

```diff
@@ -155,29 +152,19 @@
                   </v-row>
                 </v-col>
               </v-row>
-              <div
-                v-if="
-                  item &&
-                  ((item.MediaSources && item.MediaSources.length > 1) ||
-                    videoTracks.length > 0 ||
-                    audioTracks.length > 0 ||
-                    subtitleTracks.length > 0)
-                "
-                class="mt-2"
-              >
+              <div v-if="item && item.MediaSources" class="mt-2">
```

Comment:
Why the other conditions were removed?

Reply:
I wanted to show the different selectors even when there was nothing to pick, as having "No video/audio/sub available" is better than getting nothing at conveying the fact that the media has no tracks.

Reply:
Though I should add a placeholder in the media source vselect

Reply:
And disable it in that case, so the "No available X" is always shown but interaction with the v-select is completely disabled.